[SP-5012] Postgres terminate long running sessions (#47)
stackpulse/playbooks: postgres/long-running-sessions/README.md

[![Import to StackPulse](../../images/open_in_stackpulse.svg)](https://app.stackpulse.io/playbook/create?tab=playbook#https://github.com/stackpulse/playbooks/blob/master/postgres/long-running-sessions/playbook.yaml)

> This playbook collects all non-idle long running sessions from PostgresSQL instance, sends it to Slack recipients and interactively let the user the ability to terminate sessions.

Reviewer: ```suggestion
This playbook collects all non-idle long running sessions from a PostgresSQL instance, sends them to Slack and interactively allows the user to terminate the sessions.
```

## Your benefits

- Quickly identify long running sessions which can cause performance overheads
- Initiate triage by finding the applications which running the long sessions queries
- Provide information from production PostgresSQL instance without having to provide multiple people direct access to production
- Assist engineers to handle production issues by getting wider context regardless of their PostgresSQL expertise level

> - Easily terminate chosenn sessions directly from Slack

Reviewer: ```suggestion
- Easily terminate chosen sessions directly from Slack
```

## Your experience

> ![slack_screenshot](../../images/postgres_terminate_long_running.svg)

Reviewer: where is the image?
